packages/compass-preferences-model/src/preferences.ts

```diff
@@ -388,6 +390,27 @@ const nonUserPreferences: Required<{
       long: 'Allow specifying command-line flags that Compass does not understand, e.g. Electron or Chromium flags',
     },
   },
+  positionalArguments: {
+    type: 'array',
+    required: false,
+    ui: false,
+    cli: true,
+    global: false,
+    description: {
+      short:
+        'Specify a Connection String or Connection ID to Automatically Connect',
+    },
+  },
+  file: {
```

Comment:
I might be totally missing it, but where is `file` added to the preferences on startup?
I see `positionalArguments` here: https://github.com/mongodb-js/compass/pull/3528/files#diff-06a7ccc654153e56e5f96f96cbd04b2fb5f73b79ad7fec92af0c62f432c87cc7R124

How would you feel about a more descriptive name for this that brings in its connection implications? Maybe `connectionsFile`?

Reply:
> I might be totally missing it, but where is `file` added to the preferences on startup?

About 10 lines above the one you linked, parsed from yargs :)

> How would you feel about a more descriptive name for this that brings in its connection implications? Maybe `connectionsFile`?

@alenakhineika Any thoughts? The tech design says `--file` but I'm happy to pick something else. (It should probably be singular here, though, i.e. `connectionFile` instead of `connectionsFile`.)

Reply:
I am ok with just `--file` but if this is confusing we can go with `--connectionFile` I guess.